clients/src/test/java/org/apache/kafka/clients/producer/internals/RecordAccumulatorTest.java

```diff
@@ -431,6 +434,36 @@ public void testAwaitFlushComplete() throws Exception {
         }
     }
 
+    @Test
+    public void testAwaitFlushCompleteShouldWaitForSplittedBatch() throws Exception {
+        int expectedSplittedBatches = 2;
+        int batchSize = 1025;
+        boolean isSplittedBatch = true;
```

Comment:
nit: `isSplittedBatch` and `isNonSplittedBatch` seem not affecting readability and could be removed.

Reply:
Done